src/mlpack/bindings/python/CMakeLists.txt

```diff
@@ -14,28 +14,47 @@ if (NOT BUILD_PYTHON_BINDINGS)
 endif ()
 
 # Generate Python setuptools file.
-find_package(PythonInterp)
-if (NOT PYTHON_EXECUTABLE)
-  not_found_return("Python not found; not building Python bindings.")
-else ()
-  message(STATUS "Found Python: ${PYTHON_EXECUTABLE}")
-endif ()
-
 # Import find_python_module.
 include(${CMAKE_SOURCE_DIR}/CMake/FindPythonModule.cmake)
-find_python_module(distutils)
+
+## We need to check here if Python is even available.  Although actually
+## technically, I'm not sure if we even need to know!  For the tests though we
+## do.  So it's probably a good idea to check.
+if (FORCE_BUILD_PYTHON_BINDINGS)
+  find_package(PythonInterp)
+  find_python_module(distutils)
+  find_python_module(Cython 0.24)
+  find_python_module(numpy)
+  find_python_module(pandas 0.15.0)
+  if (NOT PYTHON_EXECUTABLE OR NOT PY_DISTUTILS OR NOT PY_CYTHON OR NOT PY_NUMPY
+      OR NOT PY_PANDAS)
+    unset(BUILD_PYTHON_BINDINGS CACHE)
+    message(FATAL_ERROR "Could not Build Python Bindings")
```

Comment:
Wondering if we should be more specifc here, since we already know which package is missing? Not sure a user will lookup a couple of lines to see what package it was, I might be wrong.

Reply:
Hey @zoq I have tried your suggestion in 9bdd243. Let me know what do you think?